docs/development/buildenvironments.rst

```diff
@@ -1,141 +0,0 @@
-Build Environments
-==================
-
-Read the Docs uses container virtualization to encapsulate documentation build
-processes. Each build spins up a new virtual machine using our base image,
-which is an image with the minimum necessary components required to build
-documentation. Virtual machines are limiting in CPU time and memory, which aims
-to reduce excessive usage of build resources.
-
-Setup
------
-
-Build environments use `Docker`_ to handle container virtualization. To perform
-any development on the Docker build system, you will need to set up `Docker`_ on
-your host system. Setup of Docker will vary by system, and so is out of the
-scope of this documentation.
-
-Once you have Docker set up, you will need to pull down our build image. These
-images are found on our `Docker Hub repository`_, the source comes from our
-`container image repo`_.
-
-.. note:: The size of the Docker images is around 5 to 9 GB.
-
-To get started using Docker for build environments, you'll need to pull down at
-least one build image. For example, to pull down our latest image::
-
-    docker pull readthedocs/build:latest
-
-The default image used by our build servers is :djangosetting:`DOCKER_IMAGE`.
-This would be a good place to start testing as the ``latest`` version could
-operate differently. See ``DOCKER_IMAGE`` below for setting this configuration
-option.
-
-After this image is downloaded, you can update your settings to use the new
-image -- see `Configuration`_.
-
-.. _`Docker`: https://www.docker.com
-.. _`Docker Hub repository`: https://hub.docker.com/r/readthedocs/build/
-.. _`container image repo`: https://github.com/readthedocs/readthedocs-docker-images
-
-Configuration
--------------
-
-There are several settings used to configure usage of virtual machines:
-
-DOCKER_ENABLE
```

Comment:
It seems we are losing these docs? We should probably migrate these to the settings docs.

Reply:
I didn't find them relevant, that's why I delete this page completely. On the other hand, I don't think that using `DOCKER_ENABLED=False` works. I want to remove all mentions to settings that we don't normally use and/or we don't officially support. All of these just confuses users/contributors, unfortunately.

I moved `DOCKER_LIMITS` to the settings page because is still useful for local development and, in fact, a user needed/asked about it some weeks ago.

Reply:
(This applies to `DOCKER_ENABLED` since you mentioned that perhaps it doesn't work - I get that there are other variables that do their job, but we want to avoid confusing users anyway)

Reply:
We _are_ using `DOCKER_ENABLED`. However, the only value that works is `True`.

Some time ago, we used to support `DOCKER_ENABLED=False` and in that case we didn't use Docker to build documentation but we used the host packages (latex, pip, python, etc). However, using `False` now is completely broken and it confuses contributors.

IMHO, removing it from the documentation is the first step. The next one is remove all the code for the case when it's `False`; but it's outside the scope of this PR.

Reply:
We should at least open a ticket with a TODO removing this setting, then.